monitoring/evidently/data-quality-report/data-quality-report-component.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kfp.components import create_component_from_func, InputPath, OutputPath"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASE_IMAGE = \"quay.io/ibm/kubeflow-notebook-image-ppc64le:latest\"\n",
+    "\n",
+    "def data_quality_report(\n",
+    "    cur_path: InputPath('CSV'),\n",
+    "    output_path: OutputPath('HTML'),\n",
+    ") -> None:\n",
+    "\n",
+    "    import pandas as pd\n",
```

Comment:
Add documentation to describe this methods (this is helpful for users who want to use this component; some Kubeflow tools like Elyra display this documentation). Use this as template: https://github.com/lehrig/kubeflow-ppc64le-components/blob/main/model-building/monitor-training/monitor-training.ipynb

Reply:
forthcoming ...